sgkit_vcf/vcf_reader.py

```diff
@@ -326,7 +326,7 @@ def vcf_to_zarr(
         A path (or paths) to the input BCF or VCF file (or files). VCF files should
         be compressed and have a .tbi or .csi index file. BCF files should have a .csi
         index file.
-    output : PathType
+    output : Union[PathType, MutableMapping[str, bytes]]
         The path to the output Zarr file.
```

Comment:
It would be a little more clear if this had a description like http://xarray.pydata.org/en/stable/generated/xarray.Dataset.to_zarr.html:

> Store or path to directory in file system.

Reply:
Good point. Fixed in https://github.com/pystatgen/sgkit-vcf/pull/4/commits/292f5599fad8174a9150a42241cf3c231ad0cbcf